dnf.spec

```diff
@@ -141,6 +142,7 @@ Obsoletes:      %{name}-yum < 5
 %package -n python3-%{name}
 Summary:        Python 3 interface to DNF
 %{?python_provide:%python_provide python3-%{name}}
+Requires:       libdnf5
```

Comment:
Is this line actually needed? As `python3-dnf` depends on `dnf-data` which now depends on `libdnf5`?

Reply:
No, it shouldn't be necessary, I'll take it out.